packages/babel-core/src/parse.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import loadConfig, { InputOptions } from "./config";
+import normalizeFile from "./transformation/normalize-file";
```

Comment:
I reused `normalizeFile` and `normalizeOptions` from the transformation utils to avoid duplication of code and to make sure we're processing everything the same way. Does this make sense?